scvi/metrics/clustering.py

```diff
@@ -74,3 +75,52 @@ def entropy(hist_data):
             batches[kmatrix[indices].nonzero()[1][kmatrix[indices].nonzero()[0] == i]]
         ) for i in range(100)])
     return score / 50
+
+
+def get_data(vae, data_loader, mode):
```

Comment:
Every variable is data. Let's give this function a name that will help someone who is reading the code for the first time to understand what it does.